build.sbt

```diff
@@ -19,7 +19,7 @@ resolvers ++= Seq(
   "oss releases" at "http://oss.sonatype.org/content/repositories/releases"
 )
 
-releaseSettings
+releaseCrossBuild := true
```

Comment:
Is this required? Don't think there should be anything in this that requires cross-building.